cmd/marker/main.go

```diff
@@ -108,3 +113,24 @@ func main() {
 
 	}, time.Duration(*updateInterval)*time.Second, 1.2, true, wait.NeverStop)
 }
+
+func keepAlive() {
```

Comment:
Would you please introduce a new test into https://github.com/k8snetworkplumbingwg/ovs-cni/blob/main/tests/marker_test.go verifying that the pod survives the health check and does not get restarted?